runtime-util/src/main/java/io/camunda/connector/runtime/util/feel/FeelEngineWrapper.java

```diff
@@ -76,13 +77,21 @@ private Map<String, Object> ensureVariablesMap(final Object variables) {
     return objectMapper.convertValue(variables, MAP_TYPE_REFERENCE);
   }
 
+  public <T> T evaluate(final String expression, final Object variables) {
+    return (T) toJava(evaluateInternal(expression, variables));
```

Comment:
This should not need conversion to Java, actually, since we already use the `JavaValueMapper`.
```suggestion
    return (T) evaluateInternal(expression, variables);
```